docs/modules/technical-guide/pages/user-interface/object-identifiers.adoc

```diff
@@ -0,0 +1,82 @@
+= Object Identifiers
+
+include::common:partial$_header.adoc[]
+
+Various Scout objects like xref:technical-guide:user-interface/widget.adoc[widgets], xref:technical-guide:user-interface/page.adoc[pages] or table columns may have a unique identifier.
+
+- In Scout Classic it is called `classId`.
+- In Scout JS it is called `uuid`.
+
+== Use Cases
+
+The main purpose of such an identifier is to have a stable identifier that is refactoring safe, meaning it won't change when the class or id of the object is renamed.
+This makes it possible to persist references to such Scout objects without having to migrate the persisted data when the objects are renamed.
+
+Another use case of these IDs is UI testing.
+If the url query parameter `inspector` is set to true, these ids are written to the DOM into the attribute `data-uuid`.
+
+NOTE: `inspector` is automatically set to true when the application runs in dev mode (property `scout.devMode` is true) or the url query parameter `debug` is set to true.
```

Comment:
inspector=true not only shows the uuid/classId but also the modelClass. Is this ok?

Reply:
Yes, but only if scout.urlHints.enabled is set to true